docs/configuration_hotfixes.md

```diff
@@ -0,0 +1,203 @@
+# Configuration Hotfixes
+
+This is an advanced guide for changing low level performance configuration on a cluster to hotfix an issue or test impact.
+
+Instructions for editing (add/remove/change) kernel arguments,sysfs,proc parameters and are described in this section.
+
+## Default tunings
+
+Default tunings are applied with the [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile, it is the base for creating a Tuned CR that would be detected by [cluster node tuning operator](https://github.com/openshift/cluster-node-tuning-operator)  and finally be executed by [tuned](https://github.com/redhat-performance/tuned).
+
+## Additional kernel arguments
+
+When creating a [performance profile CR](../deploy/crds/performance.openshift.io_v1alpha1_performanceprofile_cr.yaml) , a default set of kernel arguments are created from the [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile in addition to tuned generated argument and include:
+
+`nohz=on rcu_nocbs=<isolated_cores> tuned.non_isolcpus=<not_isolated_cpumask> intel_pstate=disable nosoftlockup tsc=nowatchdog intel_iommu=on iommu=pt systemd.cpu_affinity=<not_isolated_cores> isolcpus=<isolated_cores> default_hugepagesz=<DefaultHugepagesSize> hugepagesz=<hugepages_size> hugepages=<hugepages_>`
+
+> Note: isolcpus is added only when [balanceIsolated](performance_profile.md#cpu) is disabled.
+
+Additional kernel arguments could be added in the performance profile CR using the `additionalKernelArgs` field:
+
+```yaml
+apiVersion: performance.openshift.io/v1alpha1
+kind: PerformanceProfile
+metadata:
+  name: example-performanceprofile
+spec:
+  additionalKernelArgs:
+  - "nmi_watchdog=0"
+  - "audit=0"
+  - "mce=off"
+  - "processor.max_cstate=1"
+  - "idle=poll"
+  - "intel_idle.max_cstate=0"  
+...
+```
+
+> Note: These arguments will be added on top of the default arguments mentioned above. Editing these additional arguments could be done when editing the CR. 
+
+## Custom tunings 
+
+To perform hotfixes on top of the tuned [openshift-performance](https://github.com/openshift-kni/performance-addon-operators/blob/master/build/assets/tuned/openshift-node-performance) base profile, a tuned custom profile (A child profile) will be used to apply the desired changes.
+This profile will inherit the base tuned profile and override its fields where needed. 
+
+
+### Getting the current deployed tuned profile
+
+In order to apply changes we will need to get the name of the deployed tuned profile that was generated by the performance addon operator:
+
+```
+#oc describe performanceprofile <profile name> | grep Tuned
+Tuned:  <tuned namespace>/<tuned name>
+
+```
+Any tuned profile created for custom tunings will need to inherit from this tuned profile: 
+
+`include=<tuned name>`
```

Comment:
This is confusing. On line 51 you get the OpenShift name of Tuned CR. But here you use the profile tuned name! I think a step that shows how to get the proper name is missing.

Reply:
added a full example